templates/frontend/components/registrationForm.tpl

```diff
@@ -25,6 +25,7 @@
 			<label>
 				{translate key="user.firstName"}
 				<input class="form-control" type="text" name="firstName" id="firstName" value="{$firstName|escape}" maxlength="40" required>
+                                <small>{translate key="common.required"}</small>
```

Comment:
I think we'll run into a lot of styling issues with different Bootstrap themes if we try to display the full text. The display of the `<small>` tag isn't always consistent with a nice presentation alongside the form controls.

Instead, I'd like to use the `*` indicator and just put the "required" message in a hidden wrapper for screen readers. That would look like this:

```
<label>
  {translate ...}
  <span class="form-control-required">*</span>
  <span class="sr-only">{translate key="common.required"}</span>
  <input ...>
</label>
```

We can then include some consistent styles for the `.form-control-required`:

```
.form-control-required {
  color: red;
}
```

This CSS code should be added to `/styles/bootstrap.less`. We're putting custom styles there so that the base library files remain untouched. That makes it easier to update them down the road if necessary.

Reply:
On second thought, instead of `color: red;`, we should use `color: @brand-danger;`.